### DISCUSSION TITLE
High resolution SRTM not linked to zoom evel

### DISCUSSION THREAD

**Discussion by tuomaspap** [domlysz/BlenderGIS#483] [Ideas]
Hi , Ive been using the addon to create assets of islands for a game. I see the higher the zoom level the more precise the SRTM data  when applied. I was wondering is there a way to still get the more accurate SRTM data at a lesser zoom level ?

Currently to get good resolution of image and srtm i need to zoom in alot and therefor have to create alot more segements which when doing larger islands is very time consuming,

**Comment by domlysz** [comment]
The SRTM data has a fixed resolution of 30 m/pixel, in others words the precison of the downloaded data is not related to the zoom level. But this image is then used as displacement texture for the plane object (see modifier stack), and the displacer rely on a subsurf to generate the needed vertices. With large object you need to increase the subsurf klevel (or add another one) to produce more vertices and increase the relief details.
